CybeReason/malop/tests/malop_connection.json

```
@@ -0,0 +1,41 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "cybereason-malop",
+        "dialect_uuid": "0de050fb-3f56-4c7a-a9b6-76bf5298a617"
+      }
+    },
+    "message": "CEF:0|Cybereason|Cybereason|1.0|5|Malop Connection Added|5|CybereasonCEFgeneratorBatchId1=58bc2665-b22f-4345-bd90-3f84be47c8b6 cs1=11.1323449861766643222 CybereasonCEFgeneratorcountry1Name=None dst=3.226.77.3 dpt=443 rt=1629500007043 cs1Label=MalopId"
+  },
+  "expected": {
+    "@timestamp": "2021-08-20T22:53:27.043000Z",
+    "message": "CEF:0|Cybereason|Cybereason|1.0|5|Malop Connection Added|5|CybereasonCEFgeneratorBatchId1=58bc2665-b22f-4345-bd90-3f84be47c8b6 cs1=11.1323449861766643222 CybereasonCEFgeneratorcountry1Name=None dst=3.226.77.3 dpt=443 rt=1629500007043 cs1Label=MalopId",
+    "observer": {
+        "vendor": "Cybereason",
+        "product": "Cybereason",
+        "version": "1.0"
+    },
+    "destination": {
+        "ip": "3.226.77.3",
+        "port": 443
+    },
+    "event": {
+        "severity": 5,
+        "code": "5",
+        "id": "58bc2665-b22f-4345-bd90-3f84be47c8b6",
+        "kind": "event",
+        "type": "info",
```

```suggestion
        "type": "info",
```

Is it always `event.type:info`


For example : `activity": {"type": "MALICIOUS_INFECTION"},`
Does not correspond to https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-type.html#ecs-event-type-info

> subset of events within a category that indicate that they are purely informational, and don’t report a state change, or any type of action